### PR TITLE
Configure spring, remove unneeded components

### PR DESCRIPTION
**Pull request by OleksiiTromsa**

### CODE REVIEW

File: src/main/java/com/dev/cinema/config/AppConfig.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.dev.cinema.dao.impl",
+        "com.dev.cinema.service.impl",
+        "com.dev.cinema.security"
+})
+public class AppConfig {
+    private final Environment environment;
+
+    public AppConfig(Environment environment) {
```
Comment (viaviaj) [inline src/main/java/com/dev/cinema/config/AppConfig.java:23]: A good practice is adding annotation @Autowired above

**Review by viaviaj — APPROVED** [review]
Good job!
I recommend add @Autowired above all constructors, where we use instead @Injection

File: src/main/java/com/dev/cinema/config/AppConfig.java (hunk 1 above)
Comment (OleksiiTromsa) [reply src/main/java/com/dev/cinema/config/AppConfig.java:23]: I thought, that it is unnecessery if we have only one constructor, Waiting for menthor's comment :)

File: src/main/java/com/dev/cinema/config/AppConfig.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.dev.cinema.dao.impl",
+        "com.dev.cinema.service.impl",
+        "com.dev.cinema.security"
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/config/AppConfig.java]:
```suggestion
        "com.dev.cinema.dao",
        "com.dev.cinema.service",
        "com.dev.cinema.security"
```

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**